src/middlewares/errorHandler.ts

```diff
@@ -5,12 +5,15 @@ import logger from '../lib/logger'
 export default function errorHandler (error: BaseError, req: Request, res: Response, next: NextFunction): void {
   const status = error.status || 500
   const message = error.message || 'Something went wrong '
+  const meta = error.meta || {}
 
-  logger.error('Error Message:', {
-    message: message,
-    statusCode: status
+  logger.error(error.message,  {
+    ...meta,
+    stack: error.stack
   })
+  res.statusCode = status
```

Comment:
Better to use same syntax like in controllers . `req.status(status).send(data)`

Reply:
fixed 36b7a9d8d0f63934c3efff975d940c17659ee1c9